spec/rubocop/cop/style/format_string_token_spec.rb

```diff
@@ -85,6 +85,10 @@
     expect_no_offenses('`echo "%s %<annotated>s %{template}"`')
   end
 
+  it 'ignores regexp' do
+    expect_no_offenses('/foo bar %u/')
```

Comment:
Let's add a test for `%r` literal as well? 🙂

Reply:
added!